src/app/charts/box-plot/pbox-plot-view/pbox-plot-view.component.ts

```diff
@@ -135,7 +135,7 @@ export class PBoxPlotViewComponent implements OnInit, OnDestroy, AfterViewInit {
 
     getGeneSymbol(): string {
         const geneSymbol = this.geneService.getCurrentInfo().hgnc_symbol
-            ?? this.geneService.getCurrentInfo().ensembl_gene_id;
+            || this.geneService.getCurrentInfo().ensembl_gene_id;
         return geneSymbol;
```

Comment:
Note that the nullish coalescing operator didn't work for us here, as we want to fall back to ENSG when hgnc is a blank string.